### PR TITLE
add http-server as dev dep. add npm script to serve sample app

### PR DESCRIPTION
**Pull request by mellogarrett**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "lint": "eslint index.js */*.jsx",
     "lint-staged": "lint-staged",
     "prettier": "prettier -l *.js */*.jsx sample/webpack.config.js",
-    "prettier:write": "prettier --write *.js */*.jsx sample/webpack.config.js"
+    "prettier:write": "prettier --write *.js */*.jsx sample/webpack.config.js",
+    "sample": "$(npm bin)/http-server ./sample"
```
Comment (filoozom) [inline package.json]: `$(npm bin)` doesn't work on all shells or Windows, shouldn't `http-server ./sample` work without it?

Comment (mellogarrett) [reply package.json]: Nah, can't just execute a local npm package like that.

**Comment by mellogarrett** [comment]
@filoozom this new npm script should work

File: package.json (hunk 1 above)
Comment (filoozom) [reply package.json]: Wait, why not? Works perfectly for `prettier`, `lint-staged` and `eslint` in this very same file. Also, just tested specifically for `http-server` and it works without any issues.

**Comment by filoozom** [comment]
Actually, it might be better to run `webpack` before `http-serve`, wouldn't it?
I feel like it would reduce confusion. Or maybe add something like `sample:build` / `sample:serve`.

**Comment by mellogarrett** [comment]
Makes sense

**Comment by porfirioribeiro** [comment]
This is really helpful when developing and testing changes.
Couldn't this work with webpack-dev-server?

**Comment by prateekbh** [comment]
@porfirioribeiro its just a rough play space, nothing too serious. feel free to add a webpack dev server. I just didnt felt the need to do it then

**Pull request merged**